spec/integrations/allocation_spec.rb

```diff
@@ -45,7 +45,7 @@
     let(:expected_allocations) do
       if RUBY_VERSION < '2.4.0'
         16
-      elsif RUBY_VERSION >= '2.4.0' && RUBY_VERSION < '2.5.0'
+      elsif RUBY_VERSION < '2.5.0'
```

Comment:
If we get to this `elsif`, we already know that `RUBY_VERSION >= 2.4.0`. This refactor was made throughout this test file.